mesh/RemesherPro.cs

```diff
@@ -130,7 +130,7 @@ public void FastestRemesh(int nMaxIterations = 25, bool bDoFastSplits = true)
             var saveMode = this.ProjectionMode;
             for (int k = 0; k < nMaxIterations - 1; ++k) {
                 if (Cancelled())
-                    break;
+                    return;
```

Comment:
lets make the method return "bool", and when it is cancelled - return false.

Reply:
Done, but there are a lot of calls to different private methods that I don't have time to cover, so only the `RemeshesPro.FastestRemesh` returns it